project4/event.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>project4</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Francois+One|Josefin+Sans:700|Scope+One" rel="stylesheet">
+
+</head>
+
+<body>
+    <div class="the_official_angelique_site_wrapper">
+        <header>
+            <a href="index.html">
```

Comment:
In most web projects, you can link back to the root or homepage via the slash:
```suggestion
            <a href="/">
```